contracts/Voter.sol

```diff
@@ -215,22 +233,26 @@ contract Voter is IVoter {
             allowedRewards[1] = tokenB;
             internalRewards[0] = tokenA;
             internalRewards[1] = tokenB;
-
+            // if one of the tokens is not base (FLOW) then add base(FLOW) to allowed rewards
             if (base != tokenA && base != tokenB) {
-              allowedRewards[2] = base;
+                allowedRewards[2] = base;
             }
         }
 
-        if (msg.sender != governor) { // gov can create for any pool, even non-Velodrome pairs
+        if (msg.sender != governor) {
+            // gov can create for any pool, even non-Velocimeter pairs
             require(isPair, "!_pool");
             require(isWhitelisted[tokenA] && isWhitelisted[tokenB], "!whitelisted");
         }
 
         address _internal_bribe = IBribeFactory(bribefactory).createInternalBribe(internalRewards);
         address _external_bribe = IBribeFactory(bribefactory).createExternalBribe(allowedRewards);
-        address _gauge = IGaugeFactory(gaugefactory).createGauge(_pool, _internal_bribe, _external_bribe, _ve, isPair, allowedRewards);
+        address _wxbribe = IWrappedExternalBribeFactory(wrappedxbribefactory).createBribe(_external_bribe);
+        address _gauge = IGaugeFactory(gaugefactory).createGauge(
+            _pool, _internal_bribe, _external_bribe, _ve, isPair, allowedRewards
+        );
 
-        IERC20(base).approve(_gauge, type(uint).max);
+        IERC20(base).approve(_gauge, type(uint256).max);
         internal_bribes[_gauge] = _internal_bribe;
         external_bribes[_gauge] = _external_bribe;
```

Comment:
should change to following:

`external_bribes[_gauge] = _wxbribe`

Reply:
I don't know why we would change this. The wrapped bribe in velodrome functions correctly without this change.